modules/oas-page-builder/src/services/pageBuilder.ts

```diff
@@ -148,7 +152,40 @@ async function getOASpec({
 
     const path = `${output}/${pageSlug}${filePathExtension}/index.html`;
     const finalFilename = normalizePath(path);
-    await redocExecutor.execute(spec, finalFilename, buildOptions);
+
+    let versionOptions: RedocVersionOptions | undefined;
+
+    if (resourceVersions && apiVersion) {
```

Comment:
There's an edge case I'm seeing where resourceVersions might "exist" but might be an empty array. I think we might want to account for that here!